python_modules/libraries/dagster-components/dagster_components/components/shim_components/asset.py

```diff
@@ -10,9 +10,9 @@
 class AssetBlueprint(Blueprint):
     def scaffold(self, request: ScaffoldRequest, params: None) -> None:
         with pushd(str(request.target_path)):
-            Path("definitions.py").write_text("""
-# import dagster as dg
+            Path("asset.py").write_text("""# import dagster as dg
```

Comment:
this should be parameterized 

I don't think this should scaffold a folder at all, just a single python file. 

How about we revert this change and do it as a follow up so we can hash that out?